### ISSUE TITLE
With HTML or HTMLLegacy/wrap=false, <pre> / <code> are replaced with <divs>

### ISSUE THREAD

**Issue by dabrahams** [rouge-ruby/rouge#1744]
According to the README: setting `wrap: false` with the `HTMLLegacy` formatter, or simply using the `HTML` formatter, is supposed to not introduce any new `<div>`s.  Apparently, though, it actually removes `<pre>/<code>` and still leaves new `<div>` tags in place.

See http://dabrahams.github.io/pages-test/ which is rendered from https://github.com/dabrahams/pages-test

**Comment by jneen** [comment]
@dabrahams Have you verified that it is not Github Pages that is adding a wrapper?

**Comment by dabrahams** [comment]
Yes, I have verified that this happens when I run Jekyll in a local docker container.

I don't know how to make rougify accept the formatter setting:

```
$ rougify -f HTMLLegacy -F wrap=false -
unknown formatter preset HTMLLegacy
```
